.github/workflows/integration-tests.yml

```diff
@@ -182,17 +192,18 @@ jobs:
       - name: List availible images
         run: |
           docker image ls
+      #Also sets BOB_BOBP_ACTIONS_OFFSET as GITHUB_ENV from check_interval parameter (check_interval + 1s)
```

Comment:
I added comment for this in https://github.com/qoollo/bob/pull/524. In this request that line should be removed